# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 418dec4ac3f9
# Full Hash: 418dec4ac3f99323de6d6957282e8d26bd44ea19
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2022-09-28 21:33:31
# Regressor Bug: 1774317
# File Overlap Count: 2
# Description:
#   Bug 1774317 - part 2: Make `TSFTextStore` support `GUID_PROP_URL` r=m_kato
#   
#   `GUID_PROP_URL` is defined here:
#   https://learn.microsoft.com/en-us/windows/win32/tsf/predefined-properties
#   
# ==============================================================================

diff -r 329666086b40 -r 418dec4ac3f9 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Wed Sep 28 11:46:59 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Wed Sep 28 11:46:59 2022 +0000
@@ -6844,6 +6844,23 @@
     value: @IS_NOT_EARLY_BETA_OR_EARLIER@
     mirror: always
 
+  # If true, TSF and TIP (IME) can retrieve URL of the document containing
+  # the focused element.  When this is set to true, Gecko exposes the spec
+  # of the URL.
+  # And Gecko exposes only "http" and "https" URLs.  E.g., "data", "blob",
+  # "file" URLs are never exposed.
+-  name: intl.tsf.expose_url.allowed
+   type: bool
+   value: true
+   mirror: always
+
+  # If true, TSF and TIP (IME) can retrieve URL of the document containing
+  # the focused element in the private browsing mode too.
+-  name: intl.tsf.expose_url_in_private_browsing.allowed
+   type: bool
+   value: false
+   mirror: always
+
 #if defined(ENABLE_TESTS)
   # If true, NS_GetComplexLineBreaks compares the line breaks produced in the
   # content process using the Uniscribe line breaker, with those from a